archetype/simple-service/src/main/resources/archetype-resources/pom.xml

```diff
@@ -82,6 +83,11 @@
                 <version>2.0.2</version>
             </dependency>
             <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
```

Comment:
Since there is only one guava jar the version can be embedded here rather than using a property.

The value of the property is for cases like guice where there are extension jars with matching versions (e.g. guice-multibindings).